packages/effects-core/src/plugins/text/text-item.ts

```
@@ -170,46 +176,80 @@ export class TextComponent extends MaskableGraphic {
 
+      if (charCountInLine > 0) {
+        x += letterSpace;
+      }
+      x += textMetrics;
+      charCountInLine++;
+
+      if (IS_BREAK_CHAR(str)) {
+        lastBreakX = x;
+        countAtBreak = charCountInLine;
+      }
+    }
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)